test-app/guides/release/examples/index.md

```diff
@@ -70,3 +70,17 @@ Asterisks
 ___
 
 Underscores
+
+# Tables
+<table>
+  <thead>
+    <tr><th>Action</th><th>HTTP Verb</th><th>URL</th></tr>
+  </thead>
+  <tbody>
+    <tr><th>Find Record</th><td>GET</td><td>/posts/123</td></tr>
+    <tr><th>Find All</th><td>GET</td><td>/posts</td></tr>
+    <tr><th>Update</th><td>PUT</td><td>/posts/123</td></tr>
+    <tr><th>Create</th><td>POST</td><td>/posts</td></tr>
+    <tr><th>Delete</th><td>DELETE</td><td>/posts/123</td></tr>
+  </tbody>
+</table>
```

Comment:
Let's use Markdown table syntax as a baseline: 

```suggestion

| Action       | HTTP Verb | URL        |
|--------------|-----------|------------|
| Find Record  | GET       | /posts/123 |
| Find All     | GET       | /posts     |
| Update       | PUT       | /posts/123 |
| Create       | POST      | /posts     |
| Delete       | DELETE    | /posts/123 |
```

Reply:
To my surprise, the guides also uses HTML syntax. Let's skip this then to align with how it's used.